src/App.tsx

```diff
@@ -15,49 +18,90 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  LENGTH = 'length',
+  ALPHABET = 'alphabet',
+  DEFAULT = '',
```

Comment:
sorry for confusing, it should be in camel case
```suggestion
  Length = 'length',
```
etc